plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -746,6 +751,7 @@ private KubernetesListBuilder generateAppResources(List<ImageConfiguration> imag
 
             // Enrich descriptors
             enricherManager.enrich(builder);
+            enricherManager.enrich(builder);
```

Comment:
I added this by mistakenly. #1493 fixes this. apologies.

Reply:
No need to apologize, we also missed it ;-)